### ISSUE TITLE
Issue : HTML have mage size more than 10 MB then it is give timout error.

### ISSUE THREAD

**Issue by sumerpalgit** [frinyvonnick/node-html-to-image#164]
I have used this plugin for HTML TO image. And it is working amazing but when we have more than 10 MB image in HTML then it is give me timout error below so can you please help me on same.. I have try to resolve this issue and spen more than 2 days but did not get luck so please help.

(node:28687) UnhandledPromiseRejectionWarning: Error: Timeout hit: 30000
    at Object.<anonymous> (/home/ubuntu/sports/node_modules/puppeteer-cluster/dist/util.js:69:23)
    at Generator.next (<anonymous>)
    at fulfilled (/home/ubuntu/sports/node_modules/puppeteer-cluster/dist/util.js:5:58)
(Use `node --trace-warnings ...` to show where the warning was created)
(node:28687) UnhandledPromiseRejectionWarning: Unhandled promise rejection. This error originated either by throwing inside of an async function without a catch block, or by rejecting a promise which was not handled with .catch(). To terminate the node process on unhandled promise rejection, use the CLI flag `--unhandled-rejections=strict` (see https://nodejs.org/api/cli.html#cli_unhandled_rejections_mode). (rejection id: 1)
(node:28687) [DEP0018] DeprecationWarning: Unhandled promise rejections are deprecated. In the future, promise rejections that are not handled will terminate the Node.js process with a non-zero exit code.

If you need more details then please let me know.

**Comment by frinyvonnick** [comment]
Hello @sumerpalgit 👋 

Thank you for posting this issue. Can you share some reproduction so I can investigate it, please?

**Comment by krylovaaleksandra** [comment]
<html>
<body>


<video id="video_background" preload="auto" autoplay="true" loop="true" muted="muted" class="video-9--6_I">
--
  | <source src="https://cdn.discordapp.com/attachments/880449376957390941/889581462108639263/pososi_mudila.webm" type="video/webm" />
  | </video>


</body>
</html>

**Comment by balavarada** [comment]
Hi, Im using this module on testing API calls and I can able to reproduce in the following scenario,

I'm testing only one API - a POST HTTP and the no.of test data is around 509+. The payload is XML based and two to three parameters will be modified in each call.

After hitting around 430+ API calls, I'm getting the following error,

```

Error: Timeout hit: 30000
    at \node_modules\puppeteer-cluster\dist\util.js:69:23
    at Generator.next (<anonymous>)
    at fulfilled \node_modules\puppeteer-cluster\dist\util.js:5:58)
```

**Comment by frinyvonnick** [comment]
The pull request #189 let you pass a timeout option. It has been merged and released in [4.0.0](https://github.com/frinyvonnick/node-html-to-image/releases/tag/v4.0.0) 🚀 

It should fix your issue. I close this issue, Feel free to reopen it if it does not fixes it.
